app/index.js

```diff
@@ -349,7 +349,9 @@ app.on('ready', () => {
 
   load();
   plugins.subscribe(load);
-});
+}).catch(err => {
+  console.error('Error while loading extensions', err);
```

Comment:
`Error while loading extensions` Might be better to say: `Error while loading devtools extensions`